pkg/apis/provisioning/v1alpha5/provisioner_validation.go

```diff
@@ -52,13 +53,21 @@ func (s *ProvisionerSpec) validateTTLSecondsUntilExpired() (errs *apis.FieldErro
 	}
 	return errs
 }
+
 func (s *ProvisionerSpec) validateTTLSecondsAfterEmpty() (errs *apis.FieldError) {
 	if ptr.Int64Value(s.TTLSecondsAfterEmpty) < 0 {
 		return errs.Also(apis.ErrInvalidValue("cannot be negative", "ttlSecondsAfterEmpty"))
 	}
 	return errs
 }
 
+func (s *ProvisionerSpec) validateResourceLimits() (errs *apis.FieldError) {
+	if s.Limits.Resources == nil || len(s.Limits.Resources) == 0 {
```

Comment:
Zero is allowed, right? Nil is not.

Reply:
Yup zero is allowed - I just set memory to 0 and made sure it works.

Reply:
Oooooops I read len(s.Limits.Resources) == 0 as checking for zero. Curious how it's possible to hit this case though.